drinks_touch/webserver/webserver.py

```diff
@@ -22,7 +22,7 @@
 app = Flask(__name__)
 Compress(app)
 
-uid_pattern = re.compile("^\d+$")
+uid_pattern = re.compile("^\\d+$")
```

Comment:
Manual change, please review

Reply:
Good catch. But instead of adding another backlash for escaping here, prefix the string with `r`, like this: `r"^\d+$"`. Then the backslashes are not interpreted as escape sequences by python, but kept as they are and processed by the regex library. See also: https://stackoverflow.com/questions/4780088/what-does-preceding-a-string-literal-with-r-mean

Reply:
refactored :white_check_mark: